recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set rev = "release_2016.3.2" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "2016.3.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
```

Comment:
Are all of these packages required to build the recipe? I see a `package_check` function checking whether some of these are present, but many are marked as optional. If removing a library still allows the recipe to build, I would remove the ones that aren't explicitly required to build.